app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -1717,6 +1733,10 @@ private void onUpdateProgress(final int currentProgress,
         }
         binding.playbackLiveSync.setClickable(!isLiveEdge());
 
+        if (isLiveEdge() && getPlaybackSpeed() > 1.0f) {
```

Comment:
Then probably we can just do this:
```suggestion

        // if live edge of a livestream -> no need for sync button; playback speed must be <= 1.0
        final boolean liveEdge = isLiveEdge();
        binding.playbackLiveSync.setClickable(!liveEdge);
        if (liveEdge && getPlaybackSpeed() > 1.0f) {
```